nRF9160Master/src/BMS/BMHandler.c

```diff
@@ -69,8 +71,10 @@ void InitI2CCharger(void)
  * @return float - Voltage read from the I2C device, in volts.
  */
 
-float ReadI2CPMIC(float *pfVolt, float *pfTemp)
+ float ReadI2CPMIC(uint16_t *puPercent, float *pfTemp)
```

Comment:
why this type changed to uint16_t from float ?

Reply:
Made it percentage @adhilxavier